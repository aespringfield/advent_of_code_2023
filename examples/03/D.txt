.........................*.176...........62%...*......715.831.........*.........937....782...............*...*..../..............645........
