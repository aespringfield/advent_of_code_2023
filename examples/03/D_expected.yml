part_one: 62
